kiwi/tasks/system_build.py

```diff
@@ -189,7 +189,9 @@ def process(self):
         del manager
 
         # setup permanent image repositories after cleanup
-        setup.import_repositories_marked_as_imageinclude()
+        repositories = self.xml_state.get_repository_sections()
+        if bool(list(xr for xr in repositories if xr.get_imageinclude())):
```

Comment:
@schaefi yes that's nice, but if we return a boolean I would just call it a bit different, maybe something like `has_repositories_marked_as_imageinclude`, as it does not return a list of anything.